Counter/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Counter</title>
+    <link rel="stylesheet" href="./style.css" />
+    <script defer src="./js.js"></script>
```

Comment:
```suggestion
    <script defer src="./scripts.js"></script>
```
Rename the file, please